charts/cloudzero-prometheus-agent/README.md

```diff
@@ -78,12 +78,12 @@ See the `kube-state-metrics` [documentation](https://github.com/kubernetes/kube-
 
 | Key | Type | Default | Description |
 |-----|------|---------|-------------|
-| cloudzero.cloud_account_id | string | `nil` | Account ID of the account the cluster is running in. |
-| cloudzero.cluster_name | string | `nil` | Name of the clusters. |
+| cloudzero.cloudAccountId | string | `nil` | Account ID of the account the cluster is running in. |
+| cloudzero.clusterName | string | `nil` | Name of the clusters. |
 | cloudzero.host | string | `"api.cloudzero.com"` | CloudZero host to send metrics to. |
-| credentials.createSecret | bool | `true` | If true, a secret containing the CloudZero API key will be created using the `credentials.api_key` value. |
-| credentials.api_key | string | `nil` | The CloudZero API key to use to export metrics. Only used if `credentials.createSecret` is true |
-| credentials.secretName | string | `""` | The name of the secret that contains the CloudZero API key. Required if createSecret is false. |
+| cloudzero.createSecret | bool | `true` | If true, a secret containing the CloudZero API key will be created using the `credentials.apiKey` value. |
+| cloudzero.apiKey | string | `nil` | The CloudZero API key to use to export metrics. Only used if `credentials.createSecret` is true |
```

Comment:
I think this is still using the `credentials.` bit in the description here

Reply:
removed all references to the credentials in the description